dev/minikube/start.sh

```diff
@@ -11,7 +12,8 @@ if ! "${MINIKUBE}" status > /dev/null; then
     --iso-url "${ISO_URL}" \
     ${VM_DRIVER:+--vm-driver "${VM_DRIVER}"} \
     --extra-config=apiserver.runtime-config=settings.k8s.io/v1alpha1=true \
-    --extra-config=apiserver.enable-admission-plugins=MutatingAdmissionWebhook,PodPreset
+    --extra-config=apiserver.enable-admission-plugins=MutatingAdmissionWebhook,PodPreset \
+    ${MINIKUBE_EXTRA_OPTIONS:-}
```

Comment:
Please update the associated documentation files (`doc/dev/minikube.md` - Example of registry mirror would work, would it not ? )

Reply:
Thanks for the pointer to the location! Fixed in a5dde5de7985b72fcadfb22621c2ba0e087ce7f1

Reply:
LGTM.